src/App.test.js

```diff
@@ -1,16 +1,36 @@
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import App from './App';
 import { EntryProvider } from './Context/EntryContext';
 import { UserProvider } from './Context/UserContext';
 
 test('renders learn react link', () => {
   render(
-    <UserProvider>
-      <EntryProvider>
+    <EntryProvider>
+      <UserProvider>
         <App />
-      </EntryProvider>
-    </UserProvider>
+      </UserProvider>
+    </EntryProvider>
   );
   const linkElement = screen.getByText(/add an entry/i);
   expect(linkElement).toBeInTheDocument();
 });
+
+test('renders the actions', async () => {
+  render(
+    <EntryProvider>
+      <UserProvider>
+        <App />
+      </UserProvider>
+    </EntryProvider>
+  );
+
+  const button = screen.getByRole('button', { name: /submit/i });
+  const entryText = screen.getByPlaceholderText(/message/i);
+  const userInput = screen.getByPlaceholderText(/name/i);
+  userEvent.type(entryText, 'hello');
+  userEvent.type(userInput, 'chase');
+  userEvent.click(button);
+  const entry = await screen.findByText(/thanks for the message/i);
```

Comment:
There's no fetch happening here, so I believe the data will appear in time without the await. So just a screen.getByText should do.